cli/src/clients.rs

```diff
@@ -1,12 +1,12 @@
-use std::format;
-use std::path::PathBuf;
+use std::{format, path::PathBuf};
 
 use anyhow::Context;
-
-use crate::types::{
+use api::{
     CreateBundleUploadRequest, CreateBundleUploadResponse, CreateRepoRequest,
-    GetQuarantineBulkTestStatusRequest, QuarantineConfig, Repo,
+    GetQuarantineBulkTestStatusRequest, QuarantineConfig,
 };
+use context::repo::RepoUrlParts;
```

Comment:
Nit: This no longer aligns with how we name it in our other repos. We should be consistent.